docs/src/modules/components/Demo.js

```diff
@@ -217,11 +222,13 @@ class Demo extends React.Component {
 
   render() {
     const { classes, codeVariant, demo, demoOptions } = this.props;
-    const { anchorEl, codeOpen } = this.state;
+    const { anchorEl, codeOpen, demoHovered } = this.state;
+    const showSourceHint = false;
```

Comment:
Feature flag for showing the tooltip on hover. Disabled for now until we settled on a strategy.